front/app/components/EsriMap/utils.tsx

```diff
@@ -487,24 +487,29 @@ export const showAddInputPopup = ({
   popupContentNode,
   popupTitle,
 }: AddInputPopupProps) => {
-  goToMapLocation(esriPointToGeoJson(event.mapPoint), mapView).then(() => {
-    // Create an Esri popup
-    mapView.popup = new Popup({
-      collapseEnabled: false,
-      dockEnabled: false,
-      dockOptions: {
-        buttonEnabled: false,
-        breakpoint: false,
-      },
-      location: event.mapPoint,
-      title: popupTitle,
-    });
-    // Set content of the popup to the node we created (so we can insert our React component via a portal)
-    mapView.popup.content = popupContentNode;
-    // Close any open UI elements and open the popup
-    setSelectedInput(null);
-    mapView.openPopup();
-  });
+  // In case the map is using a different projection, we need to project the point to Web Mercator
+  const clickedPointProjected = projectPointToWebMercator(event.mapPoint);
+
```

Comment:
Only change here was first converting the "event.mapPoint" to the Web Mercator map projection, so we can use it in the rest of the code as before.